topi/python/topi/arm_cpu/injective.py

```diff
@@ -0,0 +1,34 @@
+# pylint: disable=invalid-name, unused-variable
```

Comment:
Were these intended to be in this diff? They seem useful but should be in a separate PR right?

Reply:
Will do